### PR TITLE
Add the JSON API media type (see https://jsonapi.org/format/).

### PR DESCRIPTION
**Pull request by MrMage**

### CODE REVIEW

File: Sources/Core/MediaType.swift [hunk 1]
```diff
@@ -198,6 +198,8 @@ extension MediaType {
     public static let multipart = MediaType(type: "multipart", subType: "mixed")
     /// JSON media type.
     public static let json = MediaType(type: "application", subType: "json", parameters: ["charset": "utf-8"])
+    /// JSON API media type (see https://jsonapi.org/format/).
+    public static let jsonApi = MediaType(type: "application", subType: "vnd.api+json", parameters: ["charset": "utf-8"])
```
Comment (tanner0101) [inline Sources/Core/MediaType.swift]: Acronyms that are commonly capitalized in English should be capitalized in Swift: `jsonAPI`.

**Review by tanner0101 — CHANGES_REQUESTED** [review]
Thanks, just one small nit.

Also, please add here if you could for Vapor 4: https://github.com/vapor/http/blob/master/Sources/HTTPKit/Headers/HTTPMediaType.swift

File: Sources/Core/MediaType.swift (hunk 1 above)
Comment (MrMage) [reply Sources/Core/MediaType.swift]: That's actually how I prefer it, I somehow thought it was not common in Swift :-) Done.

**Review by tanner0101 — APPROVED** [review]

**Comment by penny-coin** [comment]
Hey @MrMage, you just merged a pull request, have a coin! 

You now have 18 coins.

**Pull request merged**